TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Movement/AutonomousRoute.java

```diff
@@ -0,0 +1,18 @@
+package org.firstinspires.ftc.teamcode.Movement;
+
+import java.util.List;
+
+public class AutonomousRoute {
+    public List<? extends Route> routeItems;
+    public int pauseMS;
```

Comment:
Generally preferable to provide get/set methods rather than directly accessing members.

Reply:
Obvious C# programmer writing Java is obvious.